frontend/src/pages/SendMetadata/MetadataForm.jsx

```diff
@@ -105,11 +138,19 @@ export const MetadataForm = () => {
 
   const handleSubmit = async (event) => {
     event.preventDefault();
+    setLoading(true);
     const { token } = userSelector.user;
+
+    const newState = { ...state };
+
+    if (newState.releaseState === 4 && (!newState.url || newState.url === '')) {
```

Comment:
What does this do? Does it set the url to null if releaseState is 4? Maybe it should do it if releaseState != 1?

Reply:
I suppose  😩